src/guides/v2.3/architecture/archi_perspectives/ABasics_intro.md

```diff
@@ -8,12 +8,12 @@ Magento incorporates the core architectural principles of object-oriented, PHP-b
 
 The following discussion focuses on how these topics apply directly to Magento:
 
-*  Magento technology stack
-*  Magento View Model
-*  Extensibility
-*  Modularity
+*  [Magento technology stack]({{page.baseurl}}/architecture/tech-stack.html)
+*  [Magento View Model]({{page.baseurl}}/extension-dev-guide/view-models.html)
+*  [Extensibility]({{page.baseurl}}/architecture/extensibility.html)
+*  [Modularity]({{page.baseurl}}/architecture/extensibility.html)
 *  Event-driven architecture
```

Comment:
We may use the following topic as a reference for the event-driven architecture.

https://devdocs.magento.com/guides/v2.3/extension-dev-guide/events-and-observers.html